src/detections/table/pagination/index.jsx

```diff
@@ -1,5 +1,18 @@
 import React from 'react'
 
 import { TablePagination } from '@material-ui/core'
+import { useChangePage } from 'detections/table/pagination/hooks/use-change-page'
+import { useChangeRowsPerPage } from 'detections/table/pagination/hooks/use-change-rows-per-page'
+import { useContext } from 'detections/context'
+import { useSelector } from 'react-redux'
+import { get } from 'lodash'
 
-export const Pagination = () => <TablePagination component={'div'} />
+export const Pagination = () => {
+  const { pageIndex, pageSize } = useContext()
+  const count = useSelector(({ detections }) => get(detections, 'get.data.totalCount', []))
+
+  return <TablePagination component={'div'} count={count} page={pageIndex}
```

Comment:
Let's also extract these props to a separate hook, since there's so many of them.

Reply:
Done